### ISSUE TITLE
Breaking changes in error page

### ISSUE THREAD

**Issue by brillout** [brillout/vite-plugin-ssr#661]
### Description

Moved from Discord:

> There are some changes to error pages that break my Vue project. Presumably because I have an _error.page.vue, _error.page.client.ts and _error.page.server.ts and the new error pages use a different architecture it seems.
Am I using error pages wrong, or is the current release WIP, or is a migration planned
— or should I open a bug for this? 😄

**Comment by brillout** [comment]
@[discord]schneefux What's the error? Yes, there are some WIP going on but these shouldn't affect current apps, so it's a bug if it does.

**Comment by schneefux** [comment]
I created a reproduction here: https://github.com/schneefux/vps-errorpage-reproduction

It is based on the vue-full example, but it adds `_error.page.client.ts` and `_error.page.server.ts` which in this case are copies of the default files. 

On dev or build, it outputs this error:
```js
Error: [vite-plugin-ssr@0.4.86][Wrong Usage] Only one `_error.page.js` is allowed. Found several: /renderer/_error /renderer/_error /renderer/_error
    at getErrorPageId (/Users/timo/projekte/vps-errorpage-reproduction/node_modules/vite-plugin-ssr/dist/cjs/shared/route/error-page.js:15:29)
    at prerender404Page (/Users/timo/projekte/vps-errorpage-reproduction/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/renderPageContext.js:98:52)
    at prerender404 (/Users/timo/projekte/vps-errorpage-reproduction/node_modules/vite-plugin-ssr/dist/cjs/node/prerender/runPrerender.js:446:71)
```

The reproduction works fine if you downgrade, e.g. `yarn add vite-plugin-ssr@v0.4.78`.

My use case is that I built an error page which loads only the bare minimum and none of the plugins and business components of the main app (which could have triggered the error during SSR).

**Comment by brillout** [comment]
I'll release the fix later today.

**Comment by brillout** [comment]
Fix released in `0.4.87`.
